packages/cli/src/commands/run.ts

```diff
@@ -121,6 +122,7 @@ export async function run(packages: PackageSpecification[], options: RunOptions)
       }
 
       buildOutputs.push({ pkg, outputs });
+      provider.artifacts = _.merge(outputs, provider.artifacts);
```

Comment:
@dbeal-eth I am not pretty sure it won't cause any problem. While till now all scenarios I have run have included a single package, in case we have multiple packages it may cause an issue!
Besides, I have solved the issue here
https://github.com/usecannon/cannon/blob/20327907955ff3ed0fbaaac5f83c86da84dab3bc/packages/cli/src/interact.ts#L87
This was developed before but has been merged after you made this PR.

Reply:
ok as long as you confirm it has been fixed elsewhere I will go head and remove this line.